docs/repositories.md

```diff
@@ -47,14 +49,14 @@ Note that if someone else wants to commit to your recently created OpenSAFELY re
 
 ### `README.md`
 
-This file introduces your project, helps people understand the purpose of the repo, and how to navigate and use the code. 
+This file introduces your project, helps people understand the purpose of the repo, and how to navigate and use the code.
 
-Any new repo will include a template README. 
-It should be updated to briefly describe the intended study and to reflect the repo's current status as a _work in progress_. 
+Any new repo will include a template README.
+It should be updated to briefly describe the intended study and to reflect the repo's current status as a _work in progress_.
 The README should also be updated frequently to reflect changes to the repo, and should certainly be up-to-date before publication of any research outputs.
```

Comment:
I wonder should we give examples of one or two repos as best practice?

Reply:
That's a good idea. We have another ticket to [update the README](https://github.com/opensafely/research-template/issues/92) instructions, I'll add the suggestion there. 

The change here is just to remove some whitespace at the end of the lines, otherwise it wouldn't pass the tests.